### DISCUSSION TITLE
Who's using @kingstinct/react-native-healthkit?

### DISCUSSION THREAD

**Discussion by robertherber** [kingstinct/react-native-healthkit#109] [General]
I'm curious who is using @kingstinct/react-native-healthkit out in the wild and what you're building! :)

If you, your company, or a company you know about is using this library somewhere could you share it here?

**Lifeline by Kingstinct - a Biohacking Platform**
Website/app link(s): [getlifeline.app](https://getlifeline.app/)
Status: Closed Beta Testing, sign up if you're interested in jumping on the Public Beta!
Similar integrations/libraries used: [react-native-health-connect](https://github.com/matinzd/react-native-health-connect), Fitbit API, Withings API

**Comment by sfkaos** [comment]
We are using it. Stealth health platform

**Comment by antoinedelp** [comment]
Hi, we are considering moving to your package! My question is: is there a way to detect / filter out manually added data? This is essential for us as we do not want users to cheat by adding steps. Is there a flag or setting to allow not fetching manual data?
